homeassistant/components/totalconnect/config_flow.py

```diff
@@ -27,6 +28,9 @@ def __init__(self):
     async def async_step_user(self, user_input=None):
         """Handle a flow initiated by the user."""
         errors = {}
+        data_schema = vol.Schema(
```

Comment:
I'd move this back down to where it were so we don't create the schema unless we need to.

Reply:
done